src/wizards/tokenSwapDealWizard/components/daoStageContent/daoStageContent.ts

```diff
@@ -1,7 +1,32 @@
 import { IDAO } from "entities/DealRegistrationTokenSwap";
```

Comment:
General comment on this file: Could you go through the whole file and add 
- types to vars
- return types to methods/functions